src/components/stage/StageEntity.tsx

```
@@ -0,0 +1,175 @@
+import { memo, useLayoutEffect, useMemo, useRef, useState } from "react";
+import {
+  type StageLodLevel,
+} from "../../lib/stage-lod";
+import { type StageEntityRenderModel } from "../../lib/stage-render-batch";
+import type { OfficeEntity, OfficeRun } from "../../types/office";
+
+export function statusFocusAccent(status: OfficeEntity["status"]): string {
```

ESLint

[error] Fast refresh only works when a file only exports components. Use a new file to share constants or functions between components.

(react-refresh/only-export-components)